tests/test_statistics.py

```diff
@@ -0,0 +1,55 @@
+
+from sporkfish.statistics import Statistics
+
+# TODO: add tests to check searcher is indeed incrementing statistics correctly
```

Comment:
This was part of the original ask in #99. Maybe I do it in next PR - I do think it is very important to have tests to check that searcher is incrementing the stats correctly.